Assets/Scripts/Level/TutorialTriggerGroup.cs

```diff
@@ -52,6 +52,7 @@ void UpdateTriggers()
             trigger.AddGameUI(gameUI);
             trigger.AddPlayerProgress(playerProgress);
         }
+        Instance = this;

```

Comment:
why do you need this? It's set up in Awake()

Reply:
UpdateTriggers is called multiple times, and the awake version has gameUI, playerProgress and TutorialTriggers[] set to null and 0 respectively. In order for the instance to be usable it needs to be kept current. So this update is required.